src/main/java/thaumicenergistics/common/integration/tc/EssentiaItemContainerHelper.java

```diff
@@ -608,7 +617,11 @@ else if( item instanceof IEssentiaContainerItem )
 				{
 					return AspectItemType.JarLabel;
 				}
-
+				//ItemAspect?
+				if( item instanceof ItemAspect )
```

Comment:
This will cause crash if TCNP is not loaded and dragged stack matches none of the items above. So add check for `Loader.isModLoaded("thaumcraftneiplugin")`